scripts/keymap.lua

```diff
@@ -5,12 +5,12 @@ kmp = {}
 -- Microbe Editor --
 
 kmp.undo = {"ctrl", "Z"}
-kmp.redo = {"ctrl", "R"}
+kmp.redo = {"ctrl", "Y"}
```

Comment:
Not to be nitpicking, but Ctrl-y is the standard for Windows. Most of the application I know in Linux use Ctrl-Shift-Z instead 😛 
But that's for a next feature, having this configurable 👍